.github/workflows/build.yml

```diff
@@ -72,7 +72,10 @@ jobs:
         run: mix format --check-formatted
 
       - name: ▶️ Run unit tests
-        run: set -a;source oli.env && MIX_ENV=test mix ecto.reset && mix test
+        run: |
+          set -a;source oli.env
+          MIX_ENV=test mix ecto.reset
+          mix test || if [[ $? = 2  ]]; then mix test --failed; else false; fi # https://angelika.me/2024/01/08/do-not-run-mix-test-failed/
```

Comment:
I agree this would save us some time and headaches when pushing code, but it also was the way to identify and track flaky tests.

Reply:
We do not want to turn off the mix test enforcement.

It seems like this perhaps just reprints the failed tests? If that is the case and the CI still fails then this is fine and a nice addition. But we need to make sure no PR can land with failing tests, flaky or not